pkg/gadgets/advise/seccomp/tracer/tracer.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/cilium/ebpf"
 	"github.com/cilium/ebpf/link"
-	libseccomp "github.com/seccomp/libseccomp-golang"
+	syscalls "github.com/inspektor-gadget/inspektor-gadget/pkg/utils/syscalls"
```

Comment:
nit: (same in other places)

```suggestion
	"github.com/inspektor-gadget/inspektor-gadget/pkg/utils/syscalls"
```